services/web/server/tests/unit/with_dbs/02/test_projects_crud_handlers.py

```diff
@@ -393,6 +414,7 @@ async def test_list_projects_with_innaccessible_services(
         (UserRole.TESTER, status.HTTP_200_OK),
     ],
 )
+@pytest.mark.testit
```

Comment:
remove this please. the pre-commit hook should have removed it. are you sure you are running it?